apps/s_client.c

```diff
@@ -1554,6 +1558,15 @@ int s_client_main(int argc, char **argv)
         }
     }
 
+    if (localstr != NULL) {
+        if (connectstr == NULL && ! (host != NULL && port != NULL)) {
+           BIO_printf(bio_err,
+                      "%s: -local requires -connect or -host and -port\n",
+                      prog);
```

Comment:
Formally speaking why? Why shouldn't I be allowed to bind to interface of my choice and talk to implied localhost:4433? It does work. Or why not bind to 127.1.1.1? Works too...

Reply:
Good point.  Will test and amend.

Reply:
Test was rubbish, removed.